Libraries/Components/StatusBar/StatusBar.js

```diff
@@ -19,14 +19,35 @@ const processColor = require('processColor');
 
```

Comment:
Historically API and Components use two different ways and for components (ex: StatusBar) the current autogen already understands jsdocs. I had to be specific for the API docs though.